py3status/modules/wifi.py

```diff
@@ -11,9 +11,8 @@
     device: Wireless device name (default "wlan0")
     down_color: Output color when disconnected, possible values:
         "good", "degraded", "bad" (default "bad")
-    format_down: Output when disconnected (default "W: down")
-    format_up: See placeholders below
-        (default "W: {bitrate} {signal_percent} {ssid}")
+    format: Display format for this module
+        (default 'W: [{bitrate} {signal_percent} {ssid}|\?show down]')
```

Comment:
I think the whole `class Meta` needs to die because it deletes the params before the post_config_hook() that uses them